citools/dump_failures.py

```diff
@@ -144,3 +151,16 @@ def summarize_failures():
             summary_dict[manifest][test] = context
 
     return summary_dict
+
+def process_regressions_helper(regressions, regression_class):
+    # get each manifest in the regressions
+    for manifest in regressions:
+        REGRESSIONS[manifest] = regression_class
+    print("REGRESSIONS: ", REGRESSIONS)
+
+
+def process_regressions(regressions):
+    process_regressions_helper(regressions.real, "real")
+    process_regressions_helper(regressions.intermittent, "intermittent")
```

Comment:
I think we will remove `intermittent` issues in the future, think of ways to skip this, either through a cli flag, or certain criteria, or just removing the line.

Reply:
@jmaher 
we could remove that line (would probably be the better option in terms of runtime) or maybe do something like this in case we needed it later in the future?

> 
            if manifest in REGRESSIONS and REGRESSIONS[manifest]!= 'intermittent':
                  line =  REGRESSIONS[manifest]
                  print("\n" + manifest + " Regressions class: " + line)
                  print()

Reply:
I think this will work, lets remove the line and see what the printing looks like.  I am curious about runtime and how noisy the results look.

Reply:
@jmaher now it prints only unknown and real failures(if any)

Reply:
![Screenshot 2022-01-05 at 5 06 52 PM](https://user-images.githubusercontent.com/31688613/148258986-233b2967-1e5f-44be-a2a0-29fbfa87e076.png)

Reply:
great, the format has a lot of wasted space, specifically the empty brackets: [] and extra newlines.  Could you clean that up a bit?

Reply:
sure